### PR TITLE
Scale design to maximise available screen space

### PR DESCRIPTION
**Pull request by microbit-robert**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -25,7 +25,9 @@
 
 :root {
   /* Seems to fix a Safari glitch: https://github.com/microbit-foundation/python-editor-next/issues/369 */
```
Comment (microbit-robert) [inline src/App.css:27]: Will this change cause this issue to appear again? https://github.com/microbit-foundation/python-editor-next/issues/369

**Comment by microbit-robert** [comment]
Needs further work to ensure font sizes don't fall below a certain size threshold for readability. Spacing should also be reduced in some areas.

**Comment by microbit-matt-hillsdon** [comment]
I think we should seriously consider modifying the spacing scale and the font sizes in the theme before resorting to changing the root font size.

File: src/App.css (hunk 1 above)
Comment (microbit-robert) [reply src/App.css]: No longer changed in theme approach.

File: .circleci/config.yml [hunk 2]
```diff
@@ -34,7 +34,7 @@ _steps:
   install_dependencies: &install_dependencies
     run: npm ci --cache .npm-cache && sudo npm config set @microbit-foundation:registry https://npm.pkg.github.com/microbit-foundation && sudo npm i -g @microbit-foundation/website-deploy-aws@0.3.0 @microbit-foundation/website-deploy-aws-config@0.4.2 @microbit-foundation/circleci-npm-package-versioner@1
   install_theme: &install_theme
-    run: npm config set @microbit-foundation:registry https://npm.pkg.github.com/microbit-foundation && npm install --no-save @microbit-foundation/python-editor-next-microbit@0.1.0-dev.154
+    run: npm config set @microbit-foundation:registry https://npm.pkg.github.com/microbit-foundation && npm install --no-save @microbit-foundation/python-editor-next-microbit@0.1.0-percent.90.173
```
Comment (microbit-robert) [inline .circleci/config.yml:37]: Update this to the main deployment before merging.

**Comment by microbit-robert** [comment]
Sizes changes made to the default theme were originally made for testing purposes. Should they remain here or be removed?

**Comment by microbit-matt-hillsdon** [comment]
> Sizes changes made to the default theme were originally made for testing purposes. Should they remain here or be removed?

I think we should probably make matching changes.

**Comment by microbit-robert** [comment]
> > Sizes changes made to the default theme were originally made for testing purposes. Should they remain here or be removed?
> 
> I think we should probably make matching changes.

Scaling changes to the default theme have been applied -> [`aedb570`](https://github.com/microbit-foundation/python-editor-next/pull/945/commits/aedb57086b2cbb9f8aac893c9857650db35cbc6d).

**Review by microbit-matt-hillsdon — APPROVED** [review]
LGTM, subject to an updated theme package and final internal approval.

File: .circleci/config.yml (hunk 2 above)
Comment (microbit-robert) [reply .circleci/config.yml]: [`4d2fba2`](https://github.com/microbit-foundation/python-editor-next/pull/945/commits/4d2fba29cc2338a3682941937698e4b67e205053)

**Pull request merged**